services/ort-run/src/main/kotlin/OrtRunService.kt

```diff
@@ -375,13 +375,19 @@ class OrtRunService(
      * If [failIfRepoInfoMissing] is *true*, throw an [IllegalArgumentException] if the repository information
      * is missing; otherwise, return an empty [Repository] object.
      */
-    fun generateOrtResult(ortRun: OrtRun, failIfRepoInfoMissing: Boolean = true): OrtResult {
+    fun generateOrtResult(
+        ortRun: OrtRun,
+        loadAdvisorRun: Boolean = true,
```

Comment:
I like this feature to select the parts that should be loaded. This should also be reflected in the docs of this function.